### PR TITLE
LPS-181732 

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-181732

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro [hunk 1]
```diff
@@ -832,6 +861,31 @@ definition {
 		return ${response};
 	}
 
+	macro getObjectEntryIdByName {
```
Comment (yanan) [inline modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro:864]: maybe we can put this into CustomObjectAPI.macro later?

File: modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro [hunk 2]
```diff
@@ -898,6 +995,26 @@ definition {
...
+		var curl = ObjectDefinitionAPI._curlObjectEntriesRelationship(
+			en_US_plural_label = ${parent_plural_label},
+			objectEntry1 = ${parentEntryId},
+			objectEntry2 = ${childEntryId},
+			relationshipName = ${relationshipName},
+			virtualHost = ${virtualHost});
+
+		var response = JSONCurlUtil.put(${curl});
+
+		return ${response};
```
Comment (yanan) [inline modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro:1015]: Just wondering is this return necessary?

**Review by yanan — COMMENTED** [review]
Others all look good!

**Review by Hijky — APPROVED** [review]

File: modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro (hunk 2 above)
Comment (Hijky) [reply modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro:1015]: FYI, in my opinion, if we use the return response when it has some errors we can debug it more easier

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**